### PR TITLE
New abzats sprites, G17 improvements, MP15SD6.

### PR DESCRIPTION
**Pull request by Weisyl**
A full set of much better sprites for the Abzats which differentiates it
from the L6 SAW even better, some G17 improvements such as better price
balancing and the ability to add a suppressor to it with the addition of
the brand new MP15SD6 SMG, currently admin-spawnable only.

### CODE REVIEW

File: code/datums/uplink_item.dm [hunk 1]
```diff
@@ -174,11 +174,11 @@ var/list/uplink_items = list()
...
 /datum/uplink_item/dangerous/g17
-	name = "Syndicate Revolver"
-	desc = "A simple yet popular handgun chambered in 9mm. Made out of strong but lightwieght polymer. The standard magazine can hold up to 17 9mm cartridges. Not currently compatible with surpressors."
+	name = "Glock 17 Handgun"
+	desc = "A simple yet popular handgun chambered in 9mm. Made out of strong but lightwieght polymer. The standard magazine can hold up to 17 9mm cartridges. Recent upgrades have made it compatible with suppressors."
 	item = /obj/item/weapon/gun/projectile/automatic/pistol/g17
-	cost = 10
-	surplus = 15
+	cost = 14
+	surplus = 50
```
Comment (ghost) [inline code/datums/uplink_item.dm]: Why is surplus set to 50?

File: code/modules/projectiles/guns/projectile/pistol.dm [hunk 2]
```diff
@@ -23,14 +23,17 @@
 
 /obj/item/weapon/gun/projectile/automatic/pistol/g17
 	name = "Glock 17"
-	desc = "A classic 9mm handgun with a large magazine capacity.Used by security teams everywhere."
+	desc = "A classic 9mm handgun with a large magazine capacity.Used by security teams everywhere.Can be suppressed."
```
Comment (ghost) [inline code/modules/projectiles/guns/projectile/pistol.dm:26]: Missing space after the dots.

File: code/modules/projectiles/guns/projectile/pistol.dm [hunk 3]
```diff
@@ -23,14 +23,17 @@
...
 /obj/item/weapon/gun/projectile/automatic/pistol/g17
 	name = "Glock 17"
-	desc = "A classic 9mm handgun with a large magazine capacity.Used by security teams everywhere."
+	desc = "A classic 9mm handgun with a large magazine capacity.Used by security teams everywhere.Can be suppressed."
 	icon_state = "glock17"
 	w_class = 2
+	origin_tech = "combat=3;materials=2;syndicate=3"
 	mag_type = /obj/item/ammo_box/magazine/g17
+	can_suppress = 1
+	fire_delay = 0
```
Comment (HoaxIT) [inline code/modules/projectiles/guns/projectile/pistol.dm]: No delay on firing?

**Comment by Weisyl** [comment]
None of these are good enough reasons for my PR to be totally denied, one is a typo, one isn't even a problem, and the other is just some bad balancing.

**Comment by ghost** [comment]
The reason is simple, despite having no rights over changes, i've set it to DNM since i think it's an bad thing at this right moment.
- You're adding an more powerfull revolver to 50% of the surplus crates, which imho is an major balance issue.
- You're setting fire_delay to 0, which is being used in process_fire of gun.dm ( sleep(fire_delay) )

File: code/datums/uplink_item.dm [hunk 4]
```diff
@@ -174,11 +174,11 @@ var/list/uplink_items = list()
...
 /datum/uplink_item/dangerous/g17
-	name = "Syndicate Revolver"
-	desc = "A simple yet popular handgun chambered in 9mm. Made out of strong but lightwieght polymer. The standard magazine can hold up to 17 9mm cartridges. Not currently compatible with surpressors."
+	name = "Glock 17 Handgun"
+	desc = "A simple yet popular handgun chambered in 9mm. Made out of strong but lightwieght polymer. The standard magazine can hold up to 17 9mm cartridges. Recent upgrades have made it compatible with suppressors."
 	item = /obj/item/weapon/gun/projectile/automatic/pistol/g17
-	cost = 10
-	surplus = 15
+	cost = 14
+	surplus = 20
```
Comment (ghost) [inline code/datums/uplink_item.dm]: Let me ask this once again, is there any reasoning behind making it more expensive, but increasing it's surplus value?

**Pull request closed without merging**